openwisp_users/multitenancy.py

```diff
@@ -8,7 +8,23 @@
 OrganizationUser = load_model('openwisp_users', 'OrganizationUser')
 
 
-class MultitenantAdminMixin(object):
+class SelectOrgMixin(object):
+    def select_organization(self, request, form):
+        '''
+        When the user has access to only one organization
+        automatically set that org as the default value
+        '''
+        if request.POST:
```

Comment:
@niteshsinha17 if we are checking if the method of the `request` is `POST`, i will suggest we use `if request.method == 'POST'` instead of `if request.POST`. 

Also, if this is the case, we should not send the entire `request` object since it is only use to check the `request method`, instead, we could send the `request.method` as a parameter to the `select_organization` method

Reply:
It sounds good. I will change it.

Reply:
But I think check should be done inside select_organization method, it will prevent writing check line multiple times in future if this method is used somewhere else. Or if in future if we want to do something with request in the select_organization function then we will need to change check everywhere where this function is being used. I think this is not good of coding. What do you think?

Reply:
@niteshsinha17 what I meant was that `select_organization` should be called like this: `selecte_organization(request.method, form)` and in the `select_organization` method definition, you can do the check with `if method == 'POST'`. But It looks cleaner to just send `request` to the method. But please check for the method the way i describe. you can read more about why from [django docs](https://docs.djangoproject.com/en/3.1/ref/request-response/#django.http.HttpRequest.POST). 


So in conclusion, you could leave the function call the way it is now but in your method definition, you should do;
```
def select_organization(request, form):
    if request.method == 'POST':
        return
   ...
```

Reply:
I agree with your points. 😃